gnocchi/carbonara.py

```diff
@@ -331,8 +331,8 @@ def unserialize(cls, data, block_size, back_window):
 
     def serialize(self):
         # NOTE(jd) Use a double delta encoding for timestamps
-        timestamps = numpy.insert(numpy.diff(self.ts.index),
-                                  0, self.first.value)
+        timestamps = numpy.insert(numpy.diff(self.ts.index), 0,
+                                  int(datetime64_to_epoch(self.first) * 10e8))
```

Comment:
maybe use self.first.astype('timedelta64[ns]')? so we don't need to convert to timedelta64[s] and then multiply by 10e8